src/v2/styles/AITypingIndicatorView/AITypingIndicatorView-layout.scss

```diff
@@ -0,0 +1,5 @@
+@use '../utils';
```

Comment:
I believe we do not need this import as nothing from utils is used here

Reply:
Serves me right for copy pasting CSS files :D Thanks, will take care of it in a bit !